bm-persona/Assets/Occupancy/Occupancy.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Occupancy.swift
```

Comment:
I don't think this should be under /Assets/ right

Reply:
yeah, any thoughts on moving it to data vs having it in a separate folder

Reply:
Data